sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -158,6 +158,9 @@ trait CheckAnalysis extends PredicateHelper {
           case g: GroupingID =>
             failAnalysis("grouping_id() can only be used with GroupingSets/Cube/Rollup")
 
+          case Alias(w: WindowFunction, _) =>
+            failAnalysis(s"Window function '$w' call requires an OVER clause.")
```

Comment:
Sorry, seems we need to remove single quotation marks:
https://github.com/apache/spark/blob/b2011a295bd78b3693a516e049e90250366b8f52/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala#L2734